<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>История</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>История:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Эксперименты:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="table_history_experiments"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTabWidget" name="tab_history_info">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tab_brief">
           <attribute name="title">
            <string>Кратко</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Быстрый просмотр:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTextEdit" name="quick_view"/>
              </item>
              <item>
               <widget class="QLabel" name="label_image">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_full">
           <attribute name="title">
            <string>Подробно</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>История экспериментов:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QTableWidget" name="table_history_tickets"/>
              </item>
              <item>
               <widget class="QPushButton" name="button_load_from_db">
                <property name="text">
                 <string>Выгрузить в csv</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttons_layout">
     <item>
      <widget class="QPushButton" name="button_load">
       <property name="text">
        <string>Загрузить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
